src/main/java/th/ac/kmitl/science/comsci/example/nameprinters/EarlyNamePrinter.java

```diff
@@ -0,0 +1,25 @@
+package th.ac.kmitl.science.comsci.example.nameprinters;
+
+import java.io.PrintWriter;
+
+public class EarlyNamePrinter implements NamePrinter {
+    @Override
+    public String getName() {
+    return
+            "<li>Bearchitect</li>\n" +
+            "<li>58050237 Chayapol Limanon</li>\n" +
+            "<li>57050294 Pichavarit Panyokaew</li>\n" +
+            "<li>58050276 Thanathat Surakhup</li>\n" +
+            "<li>58050309 Pasthiyakan Kaewpitak</li>\n" +
+            "<li>58050355 Panuwich Laoudom</li>\n" +
+            "<li>57050269 Patipon Taweechat</li>\n" +
+            "<li>57050279 Piyapon Suntikan</li>\n" +
+            "<li>57050356 Apiwoot Narunatwattana</li>\n" +
+            "<li>57050252 Nontakhon kerdmongkol</li>";
```

Comment:
This will now help because the bottleneck will be this file instead of HelloServlet so your friend cannot work parallel

Reply:
To handle a previous name submissions so I use `EarlyNamePrinter` to prevent the lose of actual input. If everyone use `NamePrinter` approach instead of hard-coded then this class can be safely removed.

Reply:
Why should you break this chunk of name to separate file to show the others to follow up?

This means that your code will handle 2 structure
1. chunk of String which is earlier written
2. new Name and ID which will separate in each file.

Reply:
@champillon is it OK to leave these name in HelloServlet and let them to reimplement it using the new approach?

Reply:
nope, you need to break down those name in classes that extends NamePrinterBase.
Do not leave this code in HelloServlet and let your friend maintain the bottle-neck code.